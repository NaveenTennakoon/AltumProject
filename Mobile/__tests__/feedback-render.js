import React from 'react';
import FEEDBACKSCREEN from '../screens/FeedbackScreen';

import renderer from 'react-test-renderer';

test('renders correctly', () => {
  const tree = renderer.create(<FEEDBACKSCREEN />).toJSON();
  expect(tree).toMatchSnapshot();
});