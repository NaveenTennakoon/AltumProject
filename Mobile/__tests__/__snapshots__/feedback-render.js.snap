// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#FFFFFF",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#000000",
        "height": 100,
        "padding": 20,
      }
    }
  >
    <Image
      source={
        Object {
          "uri": "empty",
        }
      }
      style={
        Object {
          "alignSelf": "flex-start",
          "borderColor": "white",
          "borderRadius": 63,
          "borderWidth": 4,
          "height": 80,
          "margin": 10,
          "marginBottom": 10,
          "position": "absolute",
          "width": 80,
        }
      }
    />
    <Text
      style={
        Object {
          "color": "#FFFFFF",
          "fontSize": 24,
          "fontWeight": "700",
          "marginLeft": 70,
        }
      }
    />
    <Text
      style={
        Object {
          "color": "#AAAAAA",
          "fontSize": 18,
          "fontWeight": "700",
        }
      }
    >
      Salesperson
    </Text>
  </View>
  <Text
    style={
      Object {
        "color": "#000000",
        "fontSize": 19,
        "fontWeight": "700",
        "marginHorizontal": 15,
        "marginVertical": 5,
      }
    }
  >
    Feedback
  </Text>
  <RCTScrollView
    ListEmptyComponent={[Function]}
    data={Array []}
    disableVirtualization={false}
    getItem={[Function]}
    getItemCount={[Function]}
    horizontal={false}
    initialNumToRender={10}
    keyExtractor={[Function]}
    maxToRenderPerBatch={10}
    numColumns={1}
    onContentSizeChange={[Function]}
    onEndReachedThreshold={2}
    onLayout={[Function]}
    onMomentumScrollEnd={[Function]}
    onScroll={[Function]}
    onScrollBeginDrag={[Function]}
    onScrollEndDrag={[Function]}
    removeClippedSubviews={false}
    renderItem={[Function]}
    scrollEventThrottle={50}
    stickyHeaderIndices={Array []}
    updateCellsBatchingPeriod={50}
    viewabilityConfigCallbackPairs={Array []}
    windowSize={21}
  >
    <View>
      <View
        style={
          Object {
            "backgroundColor": "#FFFFFF",
            "flex": 1,
            "justifyContent": "center",
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#aaaaaa",
              "fontSize": 20,
              "fontWeight": "600",
              "marginTop": 100,
              "textAlign": "center",
            }
          }
        >
          No Orders With Feedback
        </Text>
      </View>
    </View>
  </RCTScrollView>
</View>
`;
